docs/_includes/home/svgs.html

```diff
@@ -40,5 +40,22 @@
+      <polygon id="Stroke-13" fill="#F2778A" points="38 24 38 21 36 21 36 24"></polygon>
+      <polygon id="Stroke-15" fill="#F2778A" points="36 36 36 39 38 39 38 36"></polygon>
+      <polygon id="Stroke-17" fill="#F2778A" points="43 31 45 31 45 29 43 29"></polygon>
+      <polygon id="Stroke-19" fill="#F2778A" points="29 31 31 31 31 29 29 29"></polygon>
+      <path d="M40.5527864,0.105572809 C40.8343139,-0.0351909363 41.1656861,-0.0351909363 41.4472136,0.105572809 L45.4472136,2.10557281 C45.9411921,2.35256206 46.1414164,2.9532351 45.8944272,3.4472136 L41.8944272,11.4472136 C41.7250352,11.7859976 41.3787721,12 41,12 L33,12 C32.6212279,12 32.2749648,11.7859976 32.1055728,11.4472136 L28.1055728,3.4472136 C27.8585836,2.9532351 28.0588079,2.35256206 28.5527864,2.10557281 L32.5527864,0.105572809 C32.8343139,-0.0351909363 33.1656861,-0.0351909363 33.4472136,0.105572809 L37,1.88196601 L40.5527864,0.105572809 Z M41,2.11803399 L37.4472136,3.89442719 C37.1656861,4.03519094 36.8343139,4.03519094 36.5527864,3.89442719 L33,2.11803399 L30.3416408,3.4472136 L33.618034,10 L40.381966,10 L43.6583592,3.4472136 L41,2.11803399 Z" id="Stroke-21" fill="#F2778A"></path>
+      <path d="M41,16 C41.7919213,16 42.9645949,16.2744151 44.1851583,16.8071342 C45.711006,17.4730957 47.2022149,18.4821802 48.530828,19.8276385 C51.971768,23.3122053 54,28.6358067 54,36 L54,44 L56,44 L56,36 C56,28.1268685 53.7781547,22.2950807 49.953922,18.4223615 C48.4433854,16.8926732 46.7399978,15.7400097 44.9851855,14.9741158 C43.5253632,14.3369714 42.0853646,14 41,14 L41,16 Z" id="Stroke-23" fill="#F2778A"></path>
+      <path d="M20,38 L20,36 C20,28.1975962 22.0197769,22.8063329 25.4546847,19.4670131 C26.7669446,18.1912713 28.240019,17.2711361 29.7726779,16.6837792 C30.970063,16.2249084 32.1394179,16 33,16 L33,14 C31.8898165,14 30.478545,14.2714375 29.0569783,14.8162208 C27.2773229,15.4982336 25.5724992,16.5631278 24.0605653,18.0329869 C20.2150245,21.7715124 18,27.6839379 18,36 L18,38 L20,38 Z" id="Stroke-25" fill="#F2778A"></path>
+      <path d="M42,13 C42,12.4482847 41.5517153,12 41,12 L33,12 C32.4482847,12 32,12.4482847 32,13 C32,13.5517153 32.4482847,14 33,14 L41,14 C41.5517153,14 42,13.5517153 42,13 Z M44,13 C44,14.6562847 42.6562847,16 41,16 L33,16 C31.3437153,16 30,14.6562847 30,13 C30,11.3437153 31.3437153,10 33,10 L41,10 C42.6562847,10 44,11.3437153 44,13 Z" id="Stroke-27" fill="#F2778A"></path>
+    </g>
```

Comment:
pls make sure that this doesn't break the other svgs used in the home page, i remember that if you didn't place a particular one at the end some svgs would render weirdly.